.ci/tag-from-ref.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+tag="${1#refs/tags/}"
+
+if [ "${tag}"  = "${1}" ]; then
```

Comment:
You can delete the extra space before the `=`. Also, it's probably best to add a trailing newline to the file.